File: quote-service/src/main/java/com/vmware/tanzu/quotes/domain/IexQuote.java
package com.vmware.tanzu.quotes.domain;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import lombok.Data;

import java.math.BigDecimal;

@JsonIgnoreProperties(ignoreUnknown = true)
@Data
public class IexQuote {

    private String symbol;
    private String companyName;
    private BigDecimal open;
    private BigDecimal close;
    private Long closeTime;
    private BigDecimal high;
    private BigDecimal low;
    private BigDecimal latestPrice;
    private Long latestUpdate;
    private String latestSource;
    private BigDecimal change;
    private BigDecimal changePercent;
    private BigDecimal avgTotalVolume;
    private BigDecimal marketCap;

}

File: quote-service-dotnet/src/Models/IexQuote.cs
using System;
using Newtonsoft.Json;

public class IexQuote
{

    public string symbol { get; set; }
    public string companyName { get; set; }
        [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
    public Decimal open { get; set; }
    [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
    public Decimal close { get; set; }
    [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
    public Int64 closeTime { get; set; }
    [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
    public Decimal high { get; set; }
    [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
    public Decimal low { get; set; }
    [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
    public Decimal latestPrice { get; set; }
    [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
    public Int64 latestUpdate { get; set; }
    [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
    public string latestSource { get; set; }
    [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
    public Decimal change { get; set; }
    [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
    public Decimal changePercent { get; set; }
    [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
    public Decimal avgTotalVolume { get; set; }
    [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
    public Decimal marketCap { get; set; }

}
